pkg/storesync/storesync.go

```diff
@@ -113,17 +113,13 @@ func Sync(ctx context.Context,
 			// Handle response
 			if err != nil {
 				if err == v1alpha1.ErrKeyNotFound { // not found, soft warn
-					logrus.WithField("z-req", req.ActionRef).
-						Warnf("Skipped sync action = %d for key %s, reason: %v", req.RequestID, ref.Key, err)
+					slog.Warn("Skipped sync action", slog.Any("id", req.RequestID), slog.Any("key", ref.Key), slog.Any("err", err), slog.Any("z-req", req.ActionRef))
 				} else { // otherwise, log error
-					logrus.WithField("z-req", req.ActionRef).
-						Errorf("Failed to sync action = %d for key %s, reason: %v", req.RequestID, ref.Key, err)
+					slog.Error("Failed to sync action", slog.Any("id", req.RequestID), slog.Any("key", ref.Key), slog.Any("err", err), slog.Any("z-req", req.ActionRef))
```

Comment:
```suggestion
					slog.Error(fmt.Errorf("failed to sync action: %w", err), slog.Any("id", req.RequestID), slog.Any("key", ref.Key))
```
we can also rename `z-reg` field with `action` or fully remove it from logs and errors.

Reply:
Good idea, I have already removed them in this PR from `Info` level log messages, since it only logged memory addresses, I left them in for `error` level, to give even more information about the overall context of the error.